torchrec/csrc/dynamic_embedding/details/mixed_lfu_lru_strategy.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+#include <torchrec/csrc/dynamic_embedding/details/naive_id_transformer.h>
+#include <torchrec/csrc/dynamic_embedding/details/random_bits_generator.h>
+#include <atomic>
+#include <optional>
+#include <queue>
+#include <string_view>
+#include <vector>
+
+namespace torchrec {
+
+/**
+ * Mixed LFU/LRU Eviction Strategy.
+ *
+ * It evict infrequence elements first, then evict least recent usage elements.
+ *
+ * The frequency is record as the power number of 2. i.e., The freq_power is 1,
```

Comment:
The frequency is recorded* as the power number of 2

Reply:
fixed.